<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ESLLAP-3041" failures="1" tests="6" name="Test" time="0.128" errors="0" timestamp="18 Nov 2023 17:08:35 GMT">
  <testcase classname="interviewPrep.TestNgSeries.RetryTestUsingRetryInterface" name="app" time="0.021"/>
  <testcase classname="interviewPrep.TestNgSeries.RetryTestUsingRetryInterface" name="man2" time="0.002">
    <skipped/>
  </testcase> <!-- man2 -->
  <testcase classname="interviewPrep.TestNgSeries.RetryTestUsingRetryInterface" name="man2" time="0.003">
    <skipped/>
  </testcase> <!-- man2 -->
  <testcase classname="interviewPrep.TestNgSeries.RetryTestUsingRetryInterface" name="man2" time="0.002">
    <skipped/>
  </testcase> <!-- man2 -->
  <testcase classname="interviewPrep.TestNgSeries.RetryTestUsingRetryInterface" name="man2" time="0.001">
    <failure type="java.lang.AssertionError" message="expected to be true expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected to be true expected [true] but found [false]
at interviewPrep.TestNgSeries.RetryTestUsingRetryInterface.man2(RetryTestUsingRetryInterface.java:22)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- man2 -->
  <testcase classname="interviewPrep.TestNgSeries.RetryTestUsingRetryInterface" name="vi" time="0.001"/>
</testsuite> <!-- Test -->
